kalite/distributed/management/commands/persist.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand, CommandError
+import os
+import json
+from sqlitedict import SqliteDict
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        print("Converting...")
+        self.convert('data/khan/contents.json', 'data/khan/contents.sqlite')
+        self.convert('data/khan/exercises.json', 'data/khan/exercises.sqlite')
+
+    def convert(self, jsonurl, sqliteurl):
+        url = os.path.join(os.environ['KALITE_DIR'], jsonurl)
+        try:
+            jsonfile = open(url)

```

Comment:
Rather than hardcoding these locations, can you use these variables instead: https://github.com/aronasorman/ka-lite/blob/02357ca3b9d6ff04a03b5cc40f8983da382cfead/kalite/topic_tools/__init__.py#L41-44